go.mod

```diff
@@ -600,7 +600,7 @@ replace (
 	github.com/gravitational/teleport/api => ./api
 	github.com/julienschmidt/httprouter => github.com/gravitational/httprouter v1.3.1-0.20220408074523-c876c5e705a5
 	github.com/keys-pub/go-libfido2 => github.com/gravitational/go-libfido2 v1.5.3-teleport.1
-	github.com/microsoft/go-mssqldb => github.com/gravitational/go-mssqldb v0.11.1-0.20230331180905-0f76f1751cd3
+	github.com/microsoft/go-mssqldb => github.com/gravitational/go-mssqldb v0.11.1-0.20250523103516-3d675ad5d264
```

Comment:
Could we cut a release in our fork and use it here?

Reply:
I'll see if I can get the permissions for this.

Reply:
Go didn't like `+` in version, corrected to `v1.8.1-teleport.2` and now it is working fine.